### ISSUE TITLE
common updates

### ISSUE THREAD

**Issue by mmilaprat** [policycompass/policycompass#604]
Activate Save Buttons by default
add additional safe button at the bottom
Messages in Pop Up
Safe a copy for all components (safe original ID of resource for CC license) - "derived_from_id"
metadate field with link to original source (to apply with CC-BY-SA license of PC) (or user name)

**Comment by mmilaprat** [comment]
add the field "derived_from_id" to all entites. When user save a copy of an entity this field must be store the origianl ID

**Comment by slomo** [comment]
If we have all those common attributes (`is_draft`, `creator_path`, `derived_from_id`) we could put them into an abstract base model, so not everyone needs to implement those on their own.

**Comment by slomo** [comment]
Implementation status:

- [ ] datasetmanager (doesn't allow edit, hence not applicable?)
- [ ] storymanager
- [x] metricsmanger
- [ ] eventsmanager (not sure if applicable)
- [x] visualisationmanager
- [ ] fcmmanager
   - edit is enabled if not logged in, but comes up with error message on saving

**Comment by mohammedbahja** [comment]
@slomo 

>  fcmmanager
edit is enabled if not logged in, but comes up with error message on saving

Could you please clarify this ?

**Comment by slomo** [comment]
@mohammedbahja 

To reproduce I did the following: 

 1. Logout on the platform (stage)
 2. Click `browse`
 3. Click any causal model
 4. Click `edit`
 5. Click `save`
 6. Get exception

**Comment by mohammedbahja** [comment]
@slomo, when you say exception, do mean this error message ? 
<img width="481" alt="1" src="https://cloud.githubusercontent.com/assets/16784021/18789830/52612d70-81a4-11e6-9024-74903e54fd83.png">

If yes, what would you propose ?

**Comment by slomo** [comment]
Yes. That error message. Redirect the unautorized user to login before posting to the backend.
